helmfile/README.md

```diff
@@ -0,0 +1,105 @@
+# Graph Protocol indexer helmfile
+Graph protocol indexer could be installed in two different modes:
+* Network mode - participating in network, process requests and get commision
+* Standalone mode - private installation that would be used, managed only by owner.
+
+## Network mode
+You can deploy graph nodes in network mode. In this case you would participate in graph network. It would deploy additional graph indexer components, that are needed to participate in network.
```

Comment:
```suggestion
You can deploy graph node in network mode. In this case additional graph indexer components gets deployed, so that the node can join the graph network.
```